test/language/eval-code/direct/var-env-lower-lex-catch-non-strict.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-evaldeclarationinstantiation
```

Comment:
Couldn't hurt! Although given the use case, I'm hopeful that we may be able to simplify this information soon--see my comment here https://github.com/tc39/test262/issues/569#issuecomment-211431837